codecov.yml

```diff
@@ -26,7 +26,7 @@ coverage:
       default:
         enabled: no
 comment:
-  layout: "reach, diff, flags, files, footer"
+  layout: "reach, diff, flags"
```

Comment:
This codecov change seems unrelated? Perhaps you want to open a new PR.

Reply:
This is done.